utils/DeployedContractsHelpers.ts

```diff
@@ -150,10 +150,14 @@ export function getUser(hre: HardhatRuntimeEnvironment): Promise<SignerWithAddre
   return hre.ethers.getNamedSigner("TEST2");
 }
 
-export function getTreasury(hre: HardhatRuntimeEnvironment): Promise<SignerWithAddress> {
+export function getTreasurySigner(hre: HardhatRuntimeEnvironment): Promise<SignerWithAddress> {
   return hre.ethers.getNamedSigner("TREASURY");
 }
 
+export async function getTreasuryAddress(hre: HardhatRuntimeEnvironment): Promise<string> {
+  return hre.network.name === "mainnetFork" ? (await hre.ethers.getNamedSigner("TREASURY")).address : constants.treasuryAddresses[hre.network.name];
```

Comment:
lets reuse `getTreasurySigner(hre)` instead of repeating `hre.ethers.getNamedSigner("TREASURY"))`